File: Zina_Yeghiazaryan/python/25_05_2021/Fibonacci.py
def Fib(n):
    '''
        Fib function has one argument, which should be a natural number (n).
        The function returns the n-th Fibonacci item.
    '''
    if n == 0:
        return 0
    if n == 1:
        return 1
    return Fib(n - 1) + Fib(n - 2)

while True:
    try:
        n = int(input("Enter the number: "))
        print(str(n) + " Fibonacci item is: " + str(Fib(n - 1)))
    except ValueError:
        print("Entered wrong value. Please enter a number.")
        print()

File: Zina_Yeghiazaryan/before_bash/c++/Makefile/fibonacci.cpp
#include "functions.h"

int fib(int n) {
	if(n == 0) {
		return 0;
	}
	
	if(n == 1) {
		return 1;
	}
	
	if(n > 1) {
		int f1 = 0, f2 = 1, k;
		
		for(int i = 1; i <= n; ++i) {
			k = f1 + f2;
			f1 = f2;
			f2 = k;
		}
		return k;
	}
}
